equinix/equinix_sweeper_test.go

```diff
@@ -48,3 +48,7 @@ func sharedConfigForRegion(region string) (*config.Config, error) {
 func isSweepableTestResource(namePrefix string) bool {
 	return strings.HasPrefix(namePrefix, tstResourcePrefix)
 }
+
+func addTestSweeper() {
```

Comment:
You shouldn't need to add this method or have anything in the TestMain method. You can rename AddTestSweeper() to init() in the sweeper file and it will be run automatically and added in that manner.

Reply:
No changes required as I have moved sweeper test to internal package